src/Json.Schema.ToDotNet.Cli/Json.Schema.ToDotNet.Cli.csproj

```diff
@@ -8,6 +8,7 @@
   <PropertyGroup Label="AssemblyAttributes">
     <AssemblyTitle>JSON Schema To .NET Code Generation Tool</AssemblyTitle>
     <Description>Command line tool to generate a set of .NET classes from a JSON Schema</Description>
+    <RootNamespaceOverride>Microsoft.Json.Schema.ToDotNet.CommandLine</RootNamespaceOverride>
```

Comment:
>RootNamespaceOverride [](start = 5, length = 21)

In the VS 2017 project system, the assembly name matches the project file name by default, which is what we want (tool executable name ends in `.Cli.exe`. But the root namespace also matches the project name by default, and we _don't_ want that.

Reply:
You could also have changed the exe name to end in commandline as far as i'm concerned. but at this point, not sure it's worth the effort.

---
In reply to: [213877998](https://github.com/Microsoft/jschema/pull/34#discussion_r213877998) [](ancestors = 213877998)